api/desecapi/tests/base.py

```diff
@@ -689,7 +689,7 @@ def has_local_suffix(cls, domain_name: str):
     def create_token(cls, user):
         token = Token.objects.create(user=user)
         token.save()
-        return token
+        return token.key
```

Comment:
that will make it harder to delete tokens

Reply:
That's right. The intention was to later replace it with `token.plain` without having to change all the other tests which use the tokens. Otherwise, they would need to know the difference between the plain and the encrypted version.

Within out tests, we currently don't do deletion on the Token object itself (if we want to test deletion, we'd use the API). If we need to, we can do `Tokens.objects.get(key=TokenHashWithPlaintext(token)).delete()`*. (Looks like we should override the getter and allow it to accept `plain=...`.)

*: of course, `TokenHashWithPlaintext` implementation is also up for discussion

Reply:
deletion code would now be `Tokens.objects.get(key=Token.make_hash(token)).delete()`. We can write a manager that allows doing `Tokens.objects.get(plain=token).delete()`.